sunpy/net/tests/test_helioviewer.py

```diff
@@ -143,37 +143,3 @@ def test_overwrite_png(self, client):
         filepath_3 = client.download_png('2020/01/01', 2.4, "[SOHO,MDI,continuum,1,100]",
                                          overwrite=True)
         assert filepath_3 == filepath
-
-    def test_progress_jp2(self, client, capsys):
```

Comment:
I'm probably missing something, but why are these tests removed?

Reply:
It could be due to the existing client tests covering these test cases already: https://github.com/sunpy/sunpy/pull/4358/files#r523764332

Reply:
These were removed since they check that the progress bar of parfive was hidden and these were a mistake.